packages/backend/src/core/SequenceProcessor.ts

```diff
@@ -117,9 +121,14 @@ export class SequenceProcessor extends EventEmitter {
   private async process(): Promise<void> {
     this.logger.debug('Processing started')
 
+    let firstRun = true
```

Comment:
This variable looks a bit ugly at the first sight, but after reading this code I must say this solution is very clean.